digdag-tests/src/test/java/acceptance/td/RedshiftIT.java

```diff
@@ -58,8 +58,9 @@
     private static final String REDSHIFT_CONFIG = System.getenv("REDSHIFT_IT_CONFIG");
     private static final String SRC_TABLE = "src_tbl";
     private static final String DEST_TABLE = "dest_tbl";
-    private static final String DATA_SECHEMA = "data_schema";
-    private static final String STATUS_TABLE_SECHEMA = "status_table_schema";
+    private static final String DATA_SCHEMA = "data_schema";
```

Comment:
All the changes in this file are just for refactoring and keeping consistency with PgIT